hoodie-client/src/test/java/com/uber/hoodie/table/TestMergeOnReadTable.java

```diff
@@ -428,14 +429,16 @@ public void testRollbackWithDeltaAndCompactionCommit() throws Exception {
         dataFilesToRead.findAny().isPresent());
 
     /**
-     * Write 2 (updates)
+     * Write 2 (inserts + updates)
```

Comment:
I tried applying the test-code patch alone to understand. I expected the test case to fail since I did not apply the actual fix but the test passed. Can you cross-check to see if some assertion needs to be added.

Reply:
okay, let me check.